tests/test_classification.py

```diff
@@ -0,0 +1,59 @@
+import os
+import json
+import subprocess
+from .mock_server_test_case import MockServerTestCase
+from .utils import term_colors
+from jsonschema import validate
+
+class TestClassification(MockServerTestCase):
+    input_dir = 'test_classification/'
+    output_dir = 'test_classification_out/'
+    command = '-c'
+    test_name = 'Classification test'
+
+    def testOutputFiles(self):
+        input_files = os.listdir(os.path.join('tests/data', self.input_dir))
+        result = subprocess.run(['./send-inference-request.sh', '-s', '--host', '0.0.0.0', '-p',
```

Comment:
The `-s` is for segmentation. It might still work like this, but maybe we should add a classification case in the inference test tool as well. And we don't need to generate images in this case

Reply:
Thanks, good point! Wasn't actually sure what that was required for. I'll add one